model_zoo/mnist/customized_optimizers.py

```diff
@@ -0,0 +1,376 @@
+import horovod.tensorflow as hvd
+import tensorflow as tf 
+
+from tensorflow.python.training import optimizer
+
+
+class UpdateVariableBySizeHook(tf.train.SessionRunHook):
+    def __init__(self, updatable_tensor, total_count):
```

Comment:
Add a docstring to describe the `total_count` and `updatable_tensor`.

Reply:
Good idea. Docstring is added and do some renaming for readability.